src/main/scala/Mathematics/AbsMax.scala

```diff
@@ -0,0 +1,22 @@
+package Mathematics
+
+object AbsMax {
+
+	/**
+	    * Method returns absolute Max Element from the list
+	    *
+	    * @param listOfElements
+    	    * @return
+    	*/
+
+	def absMax(Elements : List[Int]): Int = {
+	    var length : Int = Elements.length;
+	    var max : Int = Elements(0)
```

Comment:
Scala is a functional language which favors immutability. Usually using mutable variables `var` or iterating over lists in an imperative way is not ideal.
You can achieve the same result in a functional way:
`def absMax(elements : List[Int]): Int = Math.abs(elements.maxBy(x => Math.abs(x)))`
or using your own implementation of `abs`:
`def absMax(elements : List[Int]): Int = abs(elements.maxBy(x => abs(x)))`

Reply:
done